### DISCUSSION TITLE
Rebranding Crab chain to Crab SubDAO

### DISCUSSION THREAD

**Discussion by hackfisher** [darwinia-network/darwinia#1386] [General]
This proposal is a follow-up to #1273 . 

Since the Crab chain has its own CRAB token as a governance token and has branched into a SubDAO of Darwinia, we should reflect this in its name and branding, and grant more flexibility and development opportunities to this SubDAO. Therefore, I propose renaming the Crab chain to Crab SubDAO.

The economic connection and design between Crab SubDAO and Darwinia Network still warrant exploration, and there are many valuable practices from other projects, such as MakerDAO, that are worth learning from.

**Comment by SasoLithops** [comment]
I have no objection in name change. 

But I would advise to open a new thread on where it is explained to average user as to what will happen to Ckton holders and the ones that have Crab locked for up to 3 years( me being one of them too ).
I also read #1273 and commented there as well.

**Reply by hackfisher** [reply]
No changes to Ckton so far AFAIK, they still share half staking power which the incentives come directly from treasury in Crab SubDAO(Zero yearly new issuance in Crab SubDAO) .

But since Crab is the innovative canary system, CKTON is to test and mock KTON, it could up to change by the government system by CRAB token.

CKTON, is immediately minted when locking CRAB, staking power is the starting utility of it, no the only one, this utility also depends on the development of the staking system, otherwise the staking system will not be able to evolve and upgrade.

One more thing for SubDAO, in the philosophy of MakerDAO, SubDAO are allowed to die if it lost its vitality, I agree with that.

**Reply by hackfisher** [reply]
In any case, CKTON already minted must still exist, CRAB locked should still be locked.

Maybe the deposit for minting new CKTON should be disabled(or max time shortened to less, eg. 6m or 12m) before changes to staking.), basically could mean less CRTON could be minted or RING will be locked in less time.

**Reply by SasoLithops** [reply]
Actually at this time Ckton is useless as it can not be traded anywhere. Maybe Ckton holders can be compensated in some Crab (maybe an average on last year it was trading on Snowswap) and Cktons get burned. We keep this going for six months or so, and inform at least once a month community to go and claim and then we let go of Ckton. What do you think?

**Reply by SasoLithops** [reply]
And yes I would disable minting of new Ckton.

**Reply by 2xJDubs** [reply]
Definitely think a discussion on the future utility of ckton and thus kton is warranted.  The only role it plays now is in additional staking rewards or a representation of staked token value, a derivative of the the staking promise, that can be bought and sold.  I would compare it to vKSM on Bifrost but when one sells vKSM you can no longer redeem the staked coins unlike with CRAB which is still owned by the person locking it.  Should ckton have some governance uses as well in the DAO?

**Comment by TouristD** [comment]
This doesn’t sound as rebranding only, looks like there will be a major change to how CRAB and CKTON work in the staking mechanism… What are those changes going to be exactly? I got a bunch of CKTON that I exchanged xRING for it a while back.. is it going to become useless?

**Reply by hackfisher** [reply]
I think this rebranding scope is not related to CKTON changes, kind of off-topic IMO. "Looks like" part has no basis.
